src/Take.Blip.Builder/Actions/Desk/LeavingFromDeskAction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Threading;
+using System.Threading.Tasks;
+using Take.Blip.Builder.Actions.ForwardMessageToDesk;
+using Take.Blip.Client;
+using Take.Blip.Client.Extensions.HelpDesk;
+
+namespace Take.Blip.Builder.Actions.LeavingFromDesk
+{
+    public class LeavingFromDeskAction : ActionBase<LeavingFromDeskSettings>
```

Comment:
Maybe I didn't understand the big picture yet, but the this action is not making much sense to me. Mainly because it basically does not have any logic. Ex. of usage:
```cs
var myImplementationOfILeavingFromDesk = new MyImplementationOfILeavingFromDesk();
var leavingFromDeskAction = new LeavingFromDeskAction(myImplementationOfILeavingFromDesk);
await leavingFromDeskAction.ExecuteAsync(context, settings cancellationToken);
```

is the same as:

```cs
var myImplementationOfILeavingFromDesk = new MyImplementationOfILeavingFromDesk();
await myImplementationOfILeavingFromDesk.CloseOpenedTicketsAsync(context, settings cancellationToken);
```

Why `LeavingFromDeskAction` is necessary if it does not have any logic?

Reply:
The action is called on ProcessActionsAsync method from FlowManager (on SDK), its only the process to by pass to TemplateHosting implementation rules.